drake/systems/plants/RigidBodyTree.h

```diff
@@ -358,6 +358,16 @@ class DRAKERBM_EXPORT RigidBodyTree
   void getTerrainContactPoints(const RigidBody& body, Eigen::Matrix3Xd &terrain_points) const;
 
   bool collisionRaycast(const KinematicsCache<double>& cache, const Eigen::Matrix3Xd &origins, const Eigen::Matrix3Xd &ray_endpoints, Eigen::VectorXd &distances, bool use_margins=false);
+  bool collisionRaycast(const KinematicsCache<double>& cache, const Eigen::Matrix3Xd &origins, const Eigen::Matrix3Xd &ray_endpoints, Eigen::VectorXd &distances, Eigen::Matrix3Xd &normals, bool use_margins=false);
+
+  void signedDistances(const KinematicsCache<double>& cache,
```

Comment:
add documentation here, too.  eventually the matlab binding documentation will be generated from the C++ docs (not the other way around).  and i'd prefer a different name here, too.